SPECS/python-lxml/python-lxml.spec

```diff
@@ -32,6 +33,7 @@ Python 3 version.
 
 %prep
 %setup -q -n lxml-%{version}
```

Comment:
can you use autosetup here instead if adding the %patch

Reply:
done